scripts/data_analysis/README.md

```diff
@@ -34,8 +35,33 @@ To run this script *from within the tooling folder*, based on your python distri
 ```shell
 python3 retrieve_schema_fields.py
 ```
-Outputs:
-A csv file in the [outputs](outputs) directory containing
+>Outputs:
+>A csv file in the [outputs](outputs) directory containing schemas and fields within them
+
+### Generate a Schema space diagram, output child-parent relations
+[generate_schema_diagram.py](generate_schema_diagram.py)
+There are several options for the running of this script:
+- Search for one schema
+- Search for multiple schema
+- No Search option - generate full diagram
+
+To run this script *from within the tooling folder*, based on your python distribution:
+- Search for one schema, where the argument is a schema name in a string format
+```shell
+python3 generate_schema_diagram.py arg1
```

Comment:
```suggestion
python3 generate_schema_diagram.py arg1
# E.g. python3 generate_schema_diagram.py sentence
```

I like to provide an example for shell command, could we provide one for each of the below commands? E.g. https://github.com/ministryofjustice/hmpps-integration-api/blob/main/docs/useful-commands.md#kubectl

Reply:
I will change this and remove the mathematical nomenclature of nth argument too.